src/helpers/loadOptimize.ts

```diff
@@ -0,0 +1,23 @@
+import { receiveExperiments } from '../app/featureFlags/actions';
+import { Store } from '../app/types';
+import { assertDocument } from '../app/utils';
+
+export const getOptimizeContainerByEnv = (window: Window) =>
+  window.location.hostname === 'openstax.org' ? 'OPT-NFHSM4B' : 'OPT-W65B3CP';
```

Comment:
any reason not to use the deployed_env configuration that we already have?

Reply:
won't `window` always be defined when this is called? I made this change to avoid having to do a null check for `containerId` below

Reply:
if you just do:
```suggestion
  DEPLOYED_ENV === 'openstax.org' ? 'OPT-NFHSM4B' : 'OPT-W65B3CP';
```
that doesn't change your null check requirements right?

i know there isn't _a lot_ of behind that `DEPLOYED_ENV` variable, but there is a little, and idk maybe would be a logical place to add more logic in the future, since its just a case of using one variable instead of another here i think its worth using the derived value

Reply:
makes sense